tests/installation/partitioning_raid.pm

```diff
@@ -94,8 +94,9 @@ sub set_lvm() {
     # set volume name
     send_key "alt-v";
     type_string "root";
+    assert_screen 'volumegroup-name-root', 5;
 
-    send_key $cmd{"finish"};
+    send_key $cmd{"finish"}, 1;
 
```

Comment:
I know this is not the subject of your pull request, but can you please help and get rid of all quotes in hashes you come along?